src/include/duckdb/common/enums/date_part_specifier.hpp

```diff
@@ -35,7 +35,8 @@ enum class DatePartSpecifier : uint8_t {
 	ERA,
 	TIMEZONE,
 	TIMEZONE_HOUR,
-	TIMEZONE_MINUTE
+	TIMEZONE_MINUTE,
+	JULIAN_DAY
```

Comment:
Can we explicitly number them?
Or maybe just a comment above JULIAN_DAY indicating this is where the DOUBLE parts start?